shell/common/animator_unittests.cc

```diff
@@ -92,9 +92,6 @@ TEST_F(ShellTest, VSyncTargetTime) {
   fml::TaskRunner::RunNowOrPostTask(shell->GetTaskRunners().GetUITaskRunner(),
                                     [engine = shell->GetEngine()]() {
                                       if (engine) {
-                                        // Engine needs a surface for frames to
-                                        // be scheduled.
-                                        engine->OnOutputSurfaceCreated();
                                         // this implies we can re-use the last
                                         // frame to trigger begin frame rather
                                         // than re-generating the layer tree.
```

Comment:
Do we still need this since the framework ought to handle frame scheduling now. If it is redundant, we can get rid of the `RunNowOrPostTask` call completely. We can do this in a later patch if necessary.

Reply:
(@ColdPaleLight feel free to add the `waiting for tree to go green` label if you think this is good to go, I'm holding off on adding it in case you plan to address this now)

Reply:
I'll do this in a later patch.